tests/test_repo_interactions.py

```diff
@@ -30,12 +31,12 @@ def test_chartpress_run(git_repo, capfd):
 
     # summarize information from git_repo
     sha = git_repo.commit("HEAD").hexsha[:7]
-    tag = f"0.0.1-{PRERELEASE_PREFIX}1.h{sha}"
+    tag = f"0.0.1-{PRERELEASE_PREFIX}.1.h{sha}"
     check_version(tag)
 
     # run chartpress
     out = _capture_output([], capfd)
-
+    print(out)
```

Comment:
Debugging statement remnant?

Reply:
Yes and no - it's there for debugging, but I think it should stay. When this test fails, the output should be shown so you can see what's wrong (pytest capture means it won't be shown unless it fails).